lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -276,19 +223,28 @@ def initialize_type_map(m = type_map)
             end
           end
 
-          # NOTE(joey): PostgreSQL intervals have a precision.
-          # CockroachDB intervals do not, so overide the type
-          # definition. Returning a ArgumentError may not be correct.
-          # This needs to be tested.
-          m.register_type "interval" do |_, _, sql_type|
+          # Belongs after other types are defined because of issues described
+          # in this https://github.com/rails/rails/pull/38571
+          # Once that PR is merged, we can call super at the top.
```

Comment:
can you open an issue on this repo to track this?

Reply:
https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/177